datadog_checks_base/datadog_checks/base/checks/network.py

```diff
@@ -11,19 +11,26 @@ class Status:
     UP = "UP"
 
 
+STATUS_TO_SERVICE_CHECK = {
```

Comment:
I think if we migrate properly we won't need this anymore.

Reply:
Yes, we can use AgentCheck.X directly